File: HorseRacing.Contracts/Models/Game/Requests/CheckPlayerConnectedToGame/CheckPlayerConnectedToGameRequest.cs
using HorseRacing.Contracts.Base.Requests;

namespace HorseRacing.Contracts.Models.Game.Requests.CheckPlayerConnectedToGame
{
    public class CheckPlayerConnectedToGameRequest : BaseRequest<CheckPlayerConnectedToGameRequestDto>
    {
        public CheckPlayerConnectedToGameRequest() : base() { }
        public CheckPlayerConnectedToGameRequest(CheckPlayerConnectedToGameRequestDto data) : base() { }
    }
}

File: HorseRacing.Web/src/interfaces/api/contracts/model/game/requests/check-player-connected-to-game/check-player-connected-to-game-request.ts
import type { BaseRequest } from "../../../../base/requests/base-request";
import type { CheckPlayerConnectedToGameRequestDto } from "./check-player-connected-to-game-request-dto";

export class CheckPlayerConnectedToGameRequest implements BaseRequest<CheckPlayerConnectedToGameRequestDto> {
    Data: CheckPlayerConnectedToGameRequestDto;

    constructor(data?: CheckPlayerConnectedToGameRequestDto) {
        this.Data = data || { UserId: '', GameId: ''};
    }
}
